gallery/src/components/NxModal/NxModalFormErrorExample.tsx

```diff
@@ -39,6 +47,9 @@ export default function NxModalFormErrorExample() {
                 <span>Example NxModal header with form content and error styling</span>
               </h2>
             </header>
+            <NxTile.HeaderSubtitle>
```

Comment:
This definitely shouldn't use `NxTile.HeaderSubtitle`.  Work with design to determine how the message should be shown in this case.  Updates to https://www.sketch.com/s/2393264e-1233-4909-99c8-a7dda02588cd/a/nRWjbOl will probably be necessary.

Reply:
> This definitely shouldn't use `NxTile.HeaderSubtitle`. Work with design to determine how the message should be shown in this case. Updates to https://www.sketch.com/s/2393264e-1233-4909-99c8-a7dda02588cd/a/nRWjbOl will probably be necessary.

Code updated here : 10fb738 . I will ensure Design updates sketch as well